sdks/java/nexmark/src/main/java/org/apache/beam/sdk/nexmark/sources/GeneratorConfig.java

```diff
@@ -186,6 +187,46 @@ public long estimatedBytesForEvents(long numEvents) {
            + numBids * configuration.avgBidByteSize;
   }
 
+  public int getAvgPersonByteSize() {
```

Comment:
<!--thread_id:cc_149244866_t; commit:dd2591661935e2ce1c9472d2dd9c401e3011ec9a; resolved:0-->
<!--section:context-quote-->
> **kennknowles** wrote:
> This kind of thing is a very good candidate for `@AutoValue`

<!--section:body-->
I am not sure how to apply AutoValue here. These getters delegate to a member's fields, it seems they have to be implemented manually.

I tried refactoring it further to make use of AutoValue but it feels the changes become too extensive. I would prefer to implement them in a separate PR.